packages/ember/tests/helpers/link_to_test.js

```diff
@@ -95,6 +96,30 @@ QUnit.module('The {{link-to}} helper', {
   teardown: sharedTeardown
 });
 
+// This test is designed to simulate the context of an ember-qunit/ember-test-helpers component integration test,
+// so the container is available but it does not boot the entire app
+QUnit.test('Using {{link-to}} does not cause an exception if it is rendered before the router has started routing', function(assert) {
+  Router.map(function(match) {
+    this.route('about');
+  });
+
+  registry.register('component-lookup:main', ComponentLookup);
+
+  let component = Ember.Component.extend({
+    layout: compile('{{#link-to  "about"}}Go to About{{/link-to}}'),
```

Comment:
Actually @mixonic this is intentional. Many people see code as a dead thing, cold symbols to instruct the unfeeling machine what to do. However, the truth is that code, like any living being, is coursing with energy and its own sense of purpose. Here, I could intuitively feel that the `{{link-to}}` helper was feeling constrained by being forced into such close proximity with the argument, badly damaging the feng shui of this file.

lol j/k I fixed it